src/core/types/api.ts

```diff
@@ -21,6 +21,11 @@ export type PaginationType = {
   page?: number;
 };
 
+export type DashboardPaginationType = PaginationType & {
+  includeRestricted: boolean;
+  onlyUnwatched?: boolean;
```

Comment:
`onlyUnwatched` is only for the Next-Up endpoint. Why is it included in this type when it's not used by the other endpoints? :thinking::thought_balloon:

Reply:
That's why it's optional.